webcompat/static/js/lib/wizard/utils.js

```diff
@@ -74,3 +74,26 @@ export const isSelfReport = (href, origin) => {
 
   return false;
 };
+
+/*
+  Grab the data URI portion inside of a serialized data URI
+  backgroundImage, i.e, for the following two possible strings,
+  'url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAACAAAAAYACAIAAABt)'
+  'url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAACAAAAAYACAIAAABt")'
+  we expect "data:image/ping;base64,iVBORw0KGgoAAAANSUhEUgAACAAAAAYACAIAAABt"
+  to be matched.
+
+  Note: browsers are inconsistent in quoting CSSOM serialization
+*/
+export const getDataURIFromPreview = (bgImage) => {
+  const re = /url\(['"]{0,1}(data:image\/(?:jpeg*|jpg|png|gif|bmp);\s*base64,.+)['"]{0,1}\)/;
```

Comment:
I wonder if anyone has ever uploaded a `.jpe` image... I guess I could check on the server. Just wondering out loud.